### ISSUE TITLE
Doctests failing

### ISSUE THREAD

**Issue by cchalmers** [diagrams/active#30]
```
Test suite active-doctests: RUNNING...
Active.hs:1151: failure in expression `samples 1 (cut 4 (interval 0 2 ->> always 3))'
expected: [0 % 1,1 % 1,3 % 1,3 % 1,3 % 1]
 but got: [0 % 1,1 % 1,2 % 1,3 % 1,3 % 1,3 % 1]

Active.hs:1169: failure in expression `samples 1 (cut 4 (interval 0 2 >>- always 3))'
expected: [0 % 1,1 % 1,2 % 1,3 % 1,3 % 1]
 but got: [0 % 1,1 % 1,2 % 1,3 % 1,3 % 1,3 % 1]

Active.hs:1243: failure in expression `map (numerator . getSum) (samples 1 a1)'
expected: [0,1,2,3,1,2,3]
 but got: [0,1,2,3,3,4,5,6]

Examples: 49  Tried: 48  Errors: 0  Failures: 3
Test suite active-doctests: FAIL
```

**Comment by byorgey** [comment]
Hmm, thanks!  I'll take a look.

**Comment by byorgey** [comment]
Fixed in @7c9b8e7 and @305710b.

**Comment by byorgey** [comment]
The lesson is that this needs a lot more QuickCheck tests before being released.
